pkg/supportbundle/spec.go

```diff
@@ -459,6 +481,44 @@ func getDefaultDynamicAnalyzers(app *apptypes.App) []*troubleshootv1beta2.Analyz
 
 	analyzers = append(analyzers, makeAPIReplicaAnalyzer())
 
+	clientset, err := k8sutil.GetClientset()
+	if err != nil {
+		logger.Errorf("Failed to get clientset for dynamic kurl analyzers: %v", err)
+	} else {
+		if kotsutil.IsKurl(clientset) {
+			analyzers = append(analyzers, &troubleshootv1beta2.Analyze{
+				Sysctl: &troubleshootv1beta2.SysctlAnalyze{
+					AnalyzeMeta: troubleshootv1beta2.AnalyzeMeta{
+						CheckName: "IP forwarding not enabled",
+					},
+					Outcomes: []*troubleshootv1beta2.Outcome{
+						{
+							Fail: &troubleshootv1beta2.SingleOutcome{
+								When:    "net.ipv4.ip_forward = 0",
+								Message: "IP forwarding not enabled",
+							},
+						},
+					},
+				},
+			},
+				&troubleshootv1beta2.Analyze{
```

Comment:
is this the indentation after running gofmt?

Reply:
Yes, it's weird though